centraldogma/src/test/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplierTest.java

```diff
@@ -133,4 +144,106 @@ public void testGetPropertyAbsentName() {
         PropertyDefinition<Object> missingProperty = PropertyDefinition.define("absent.value", Long.class);
         assertFalse(supplier.getProperty(missingProperty).isPresent());
     }
+
+    @Test
+    public void testRegisterWithDefaultSettings() {
+        when(centralDogma.normalizeRevision(PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD)).thenReturn(
+                CompletableFuture.completedFuture(Revision.HEAD)
+        );
+        when(centralDogma.listFiles(PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD, FILENAME)).thenReturn(
+                CompletableFuture.completedFuture(Collections.emptyMap())
+        );
+        when(centralDogma.push(
+                PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD,
+                String.format("[CentralDogmaPropertySupplier] Property file created: %s",
```

Comment:
I don't think  we need to match this argument and it will just make shit test fragile for tiny change on commit message which is mostly pointless. So let's use type matcher here, like `any(String.class)`

Reply:
I changed the code to use mockito `any(String.class)`, since a central dogma's commit message is not target of test for this PR's functionality.
https://github.com/line/decaton/pull/134/commits/f2ee11da05f8db02995aba1f609d0dd2271d680b

Reply:
Hmm, looks like this part and one in below hasn't changed yet?

Reply:
I'm sorry, I failed to update the code exhaustively.
I changed the `when` parts to use `any`: 0f2db81